src/oas/__tests__/__snapshots__/operation.test.ts.snap

```diff
@@ -15,14 +15,110 @@ Array [
             "examples": Array [],
             "mediaType": "application/json",
             "schema": Object {
-              "$ref": "#/definitions/Pet",
+              "properties": Object {
```

Comment:
Hmm does this mean that when we serialize and send the operation up to our backend to store in the database, that it'll end up getting stored fully dereferenced? 🤔

Reply:
We likely only want to resolve local references that are not nested within a "schema" object.

Reply:
> Hmm does this mean that when we serialize and send the operation up to our backend to store in the database, that it'll end up getting stored fully dereferenced? thinking

Yeah, but we're doing the same now after all.


> We likely only want to resolve local references that are not nested within a "schema" object.

hmmm, yeah, you might be right.